README.md

```diff
@@ -13,8 +13,8 @@ https://alpaca-components.now.sh/
 
 ## Installation
 - Install dependencies using `yarn`
-- Run `gulp fractal:start` to start Fractal.js development server
-- Run `gulp fractal:build` if you want to generate static files (for example to deploy them)
+- Run `gulp dev` to start Fractal.js development server
+- Run `gulp` if you want to generate static files (for example to deploy them)
```

Comment:
I think it should be `gulp build`

Reply:
After that you can update script in `package.json` to
```
"scripts": {
    "dev": "gulp dev",
    "build": "gulp build",
    "now-build": "gulp build --ci"
  },
```
so we'll be able to use `yarn dev` and `yarn build` commands as in other projects

Reply:
both will work, build is the default task

Reply:
"After change in scss file I see this list of running tasks." seems it's an issue with chokidar which used in Gulp 4 https://github.com/paulmillr/chokidar/issues/544
I did award around to ignore other file extensions until the issue get solved